platformio.ini

```diff
@@ -27,7 +27,7 @@ lib_deps =
     ESP8266mDNS
     me-no-dev/uSSDP
     DNSServer
-    WifiManager
+    https://github.com/tzapu/WiFiManager#bbe984c93b71601aad9f50f765ba9a255478bff3
```

Comment:
Even if the ESP32 version would require a specific branch, I wouldn't pin library versions to particular commits, especially for ESP8266.

Reply:
For this one, the latest Development branch commit doesn't even pass their own CI checks :( I'd feel better leaving it at this commit for now. I'll revisit in the future when hopefully they release esp32 support, then we can put the good branch/master :)